src/visitors/CyclomaticComplexityVisitor.java

```diff
@@ -0,0 +1,36 @@
+package visitors;
+
+import com.github.javaparser.ast.Node;
+import com.github.javaparser.ast.body.ClassOrInterfaceDeclaration;
+import com.github.javaparser.ast.expr.BinaryExpr;
+import com.github.javaparser.ast.stmt.IfStmt;
+import com.github.javaparser.ast.visitor.GenericVisitorAdapter;
+
+import java.util.List;
+
+public class CyclomaticComplexityVisitor extends GenericVisitorAdapter<Integer, Void> {
+
+    @Override
+    public Integer visit(ClassOrInterfaceDeclaration n, Void arg) {
+        return countDecisions(n);
```

Comment:
Shouldn't you add 1 here?
>A simpler defintion is to count the number of decisions in a method and add 1.

Reply:
MY interpretation was that for each decision you add 1. So it's basically the number of decisions, no?

Reply:
I assumed the `+ 1` was approximating for the `+No. of connected components` in the formula for cyclomatic complexity, which is usually 1.

Reply:
updated